app/controllers/main_controller.rb

```diff
@@ -160,7 +160,9 @@ def index
     ]).order('id ASC')
 
     @current_assignment = Assignment.get_current_assignment
-    @current_ta = @current_assignment.tas.first
+    if @current_assignment!=nil
```

Comment:
Prefer `!expression.nil?` over `expression != nil`.<br>Surrounding space missing for operator `!=`.